### PR TITLE
record exception in dubbo inst

### PR DESCRIPTION
**Pull request by tydhot**

### CODE REVIEW

File: instrumentation/apache-dubbo-2.7/library/src/main/java/io/opentelemetry/instrumentation/apachedubbo/v2_7/DubboTracer.java [hunk 1]
```diff
@@ -46,6 +46,9 @@ public Context startClientSpan(String interfaceName, String methodName) {
   }
 
   public void end(Context context, Result result) {
+    if (result.hasException()) {
```
Comment (iNikem) [inline instrumentation/apache-dubbo-2.7/library/src/main/java/io/opentelemetry/instrumentation/apachedubbo/v2_7/DubboTracer.java:49]: If you do this, then you don't need to handle span status below.

Comment (tydhot) [reply instrumentation/apache-dubbo-2.7/library/src/main/java/io/opentelemetry/instrumentation/apachedubbo/v2_7/DubboTracer.java:49]: yes i got it:)

**Review by iNikem — APPROVED** [review]

**Review by mateuszrzeszutek — APPROVED** [review]

**Comment by tydhot** [comment]
@iNikem @mateuszrzeszutek hi, it seemed that there is a windows smoke test failed. I have no idea about this, what should I do to fix it?

**Comment by mateuszrzeszutek** [comment]
Looks like a random failure - I reran all checks, hope they pass this time.

**Comment by tydhot** [comment]
@mateuszrzeszutek thanks!

**Pull request merged**